### ISSUE TITLE
Right aligned text in 'API reference' menu hard to read.

### ISSUE THREAD

**Issue by craigdanj** [expressjs/expressjs.com#961]
While it may look nice i feel that this right aligned menu would be easier to read it all the items were left aligned. I'm currently learning express and noticed that this was a problem for me. Just thought I'd mention.

Just my two cents but this would help UX in my opinion.

For eg:

It would be easier to read if it looked like this:

```
express()
Application
Request
Properties
req.app
req.baseUrl
req.body
req.cookies
req.fresh
req.hostname
req.ip
req.ips
req.method
req.originalUrl
req.params
req.path
req.protocol
req.query
req.route
req.secure
req.signedCookies
req.stale
req.subdomains
req.xhr
Methods
req.accepts()
req.acceptsCharsets()
req.acceptsEncodings()
req.acceptsLanguages()
req.get()
req.is()
req.param()
req.range()
Response
Router
```

**Comment by crandmck** [comment]
It's a reasonable suggestion.  Does anyone else have an opinion...?
@craigdanj You are welcome to open a PR with a fix if you want.

**Comment by dougwilson** [comment]
I don't have an opinion either way. The left hand side is standard for left to right languages like English anyway.

**Comment by craigdanj** [comment]
@crandmck  I have submitted a pull request for the same. Kindly let me know if anything else needs to be done. I've tested on mobile too to make sure it works.

--------------------------------------------------
**FYI:**  I also noticed that with the previous implementation (with the menu on the right), the menu overlaps the content in some places on smaller screens (screenshot attached below). This too has been fixed in this PR of mine since they both pertain to the menu.


![screen shot 2018-08-15 at 5 30 09 pm](https://user-images.githubusercontent.com/3023019/44147672-9de4f5a4-a0b2-11e8-9ed3-babf6e09d197.png)
